lib/kafka_ex/default_server.ex

```diff
@@ -0,0 +1,309 @@
+defmodule KafkaEx.DefaultServer do
```

Comment:
This has 0.8.2 and 0.9.0 things, until I peel them out, I want to keep it named default, it probably will end up being 0.9.0